src/components/themed/ThemedModal.tsx

```diff
@@ -63,7 +59,7 @@ export function ThemedModal<T>(props: Props<T>) {
       padding={paddingRem.map(theme.rem)}
       underlay={
         <BlurView
-          blurType={isDark ? 'light' : 'dark'}
+          blurType={theme.modalBlurType}
```

Comment:
This is different from the GUI, which means we can't share the theme JSON cleanly here. The reason we have an `isDark` flag in the GUI is because there are *several* things that depend on it, not just the blur. For instance, the battery icon and clock come from the OS, and those can either be white (for dark-mode apps) or black (for light-mode apps). Setting the `isDark` flag allows the theme to change all of these things at once. Please keep it this way.